pkgs/development/python-modules/symbiflow-fasm/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, pkgs
+, fetchFromGitHub
+, buildPythonPackage
+, textx
+}:
+
+buildPythonPackage rec {
+  pname = "symbiflow-fasm";
+  version = "0.0-47-g4857dde";
+
+  src = fetchFromGitHub {
+    owner = "SymbiFlow";
+    repo = "fasm";
+    rev = "4857dde757edd88688c2faf808774d85bdbe3900";
+    sha256 = "1za7f8slf8wvp1mfbfc3vdv61115p49k0vwngs4db6ips1qg1435";
+  };
+
+  buildInputs = [ textx ];
```

Comment:
this is a python package, it should go in `nativeBuildInputs` if it's being executed, `propagatedBuildInputs` if it's being used at runtime.

`buildInputs` would only be necessary if the package is trying to link against it (unlikely, but might be possible).